src/allotropy/allotrope/schemas/shared/definitions/definitions.json

```diff
@@ -240,8 +240,7 @@
                 "items": {
                     "$ref": "#/$defs/tDatacubeComponent"
                 },
-                "unique": true,
-                "minItems": 1
+                "unique": true
```

Comment:
NOTE: I had to remove this in order for this schema to match the cleaned schema in order for substitution to work. This value is not used in the dataclasses anyway.